### ISSUE TITLE
Unexpected behaviour with "<@context_body/>"

### ISSUE THREAD

**Issue by delatvan** [hackvertor/hackvertor#83]
Burp Suite Professional v2023.2.4
Hackvertor version 1.7.38

I'm experiencing an issue when using the <@context_body/> when nested inside other tags. If the tag is called alone, the body contents of the request are correctly displayed. However, if the tag is called nested inside a custom- or an existent-tag the <@context_body/> tag returns an empty value.

Before applying conversions:
```
POST /  HTTP/1.1
X-Content: <@_custom('<@context_body/>','id')><@/_custom>
X-Content2: <@_custom('<@get_body/>','id')><@/_custom>
X-test: <@set_body('true')><@context_body/><@/set_body>
X-test2: <@context_body/>
Content-Type: application/json
Accept: */*
Host: test.test

{ some json}
```

After applying conversions:
```
POST /  HTTP/1.1
X-Content: <@_custom('<EMPTY>','id')><@/_custom>
X-Content2: <@_custom('<EMPTY>','id')><@/_custom>
X-test: <EMPTY>
X-test2: {some json}
Content-Type: application/json
Accept: */*
Host: test.test

{ some json}
```

**Comment by hackvertor** [comment]
Fixed in v1.7.42. If you get chance to test it that would be great to see if it's working as desired cheers.

**Comment by delatvan** [comment]
Great! I'll test it and let you know the outcome.
